internal/admission/handler.go

```diff
@@ -63,6 +65,14 @@ func (h RequestHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
+
+	h.PromMetrics.RecordAdmissionCount(
+		response.Allowed,
+		fmt.Sprintf(
+			"%s.%s/%s",
```

Comment:
Should the format be `Group/version.Resource`?

Reply:
I think normally something like `kubectl describe crd telemetry.istio.io/v1alpha1` is used, right?

Reply:
In getting CRDs we use `kubectl [get|describe] crd telemetry.istio.io` (no version here) so it is reasonable to use this.